PWGGA/GammaConv/macros/AddTask_GammaCalo_pp.C

```diff
@@ -1638,6 +1639,8 @@ void AddTask_GammaCalo_pp(
   task->SetDoTHnSparse(enableTHnSparse);
   task->SetProduceTreeEOverP(doTreeEOverP);
   task->SetEnableSortingOfMCClusLabels(enableSortingMCLabels);
+  if(trainConfig == 2020 || trainConfig == 2021 || trainConfig == 2022) enablePi0Only = kTRUE;
+  task->SetDoPi0Only(enablePi0Only);
```

Comment:
indentation.

Reply:
done

Reply:
I see now that I think there should not be an indentation since the if only affects the enablePi0only flag, not the SetDoPi0only

Reply:
ahh true didn't read far enough ;)